tools/dump_kubernetes.sh

```diff
@@ -330,11 +426,29 @@ main() {
   local exit_code=0
   parse_args "$@"
   check_prerequisites kubectl
-  dump_time
-  dump_pilot
-  dump_resources
-  tap_containers "dump_logs_for_container" "copy_core_dumps_if_istio_proxy"
-  exit_code=$?
+  if [ "${CONTROL_PLANE}" = true ] ; then
+    namespaces="istio-system"
+    check_prerequisites istioctl
+    dump_time
+    dump_control_plane
+    tap_containers "dump_logs_for_container"
+    dump_istio_status
+    exit_code=$?
+  elif [ "${ONLY_CLUSTER}" = true ] ; then
```

Comment:
I don't think it should be exclusive, I was thinking it would by default dump nothing then you can add to it. like `dump.sh -u -c -x=foo-bar`?

Reply:
I thought about keeping the previous behavior by default, and that's how I ended up separating them this way.
You're probably right, it should be additive, I've no strong opinion on this one.